### PR TITLE
Update power-sector.md

### PR DESCRIPTION
**Pull request by DeepakIER**
Power generation sector has  been updated. Please review the text

### CODE REVIEW

File: docs/sectoral_coverage/power-sector.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+| **[%]**        | **[Year]**         |
+| **Year**       | **2020**           | **2030**              | **2040**       | **2050**     |            |  |  |  |
+| Onshore Wind   | 1.257              | 1.137                 | 987            | 923          | 3          | 100 | 25 | [1] |
+| Solar PV       | 703                | 395                   | 340            | 326          | 1          | 100 | 25 | [1] |
+| Biomass        | 2.037              | 1.954                 | 1.892          | 1.826        | 3.5        | 100 | 25 | [2], [3] |
+| Hydro Power    | \-                 | \-                    | \-             | 2.718        | 2          | 100 | 100 | [4], [5] |
+| Offshore-Wind  | \-                 | \-                    | \-             | 1.496        | 3          | 100 | 25 | [5] |
+| Nuclear        | \-                 | \-                    | \-             | 3.990        | 4          | 30 | 60 | [5] |
+
+References
```
Comment (LippkauFelix) [inline docs/sectoral_coverage/power-sector.md:19]:
```suggestion
### References
```

File: docs/sectoral_coverage/power-sector.md [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+In terms of economic evaluation, TIAM considers explicitly CAPEX, OPEX, efficiency and lifetime of the power plants for electricity supply. The model considers unlimited commodities/resources for renewable plants. Unlike coal or oil which needs expenses for production; renewable sources- solar radiation or wind energy are supplied at no costs. Hence, no additional expenses for fuel are considered.  As explained in time resolution section, TIAM model has ten different periods. The table below shows CAPEX costs for four different time periods. It is to explain the trend in declining CAPEX as the technologies get more mature and will be less expensive to invest in the future.
```
Comment (LippkauFelix) [inline docs/sectoral_coverage/power-sector.md:4]: https://etsap-tiam.readthedocs.io/en/latest/restrictions/potentials-renewable.html

Comment (olejandro) [reply docs/sectoral_coverage/power-sector.md:4]: @LippkauFelix should anything happen to this URL?

Comment (LippkauFelix) [reply docs/sectoral_coverage/power-sector.md:4]: Ah sorry, something happend to the text that I wrote. I would propose to add a reference to: 

> The model considers unlimited commodities/resources for renewable plants

Actually the renewables are restricted by using a capacity constraint, which is described using the linked page in the documentation.

Comment (olejandro) [reply docs/sectoral_coverage/power-sector.md]: Ok, thanks! Can add a cross reference there.

**Comment by olejandro** [comment]
@DeepakIER refs 2 and 4 are the same, while ref 5 is missing.

File: docs/sectoral_coverage/power-sector.md (hunk 2 above)
Comment (olejandro) [reply docs/sectoral_coverage/power-sector.md]: @LippkauFelix please check. Is it how you meant?

Comment (LippkauFelix) [reply docs/sectoral_coverage/power-sector.md:4]: Can't find it on the pre-rendered page.

Comment (olejandro) [reply docs/sectoral_coverage/power-sector.md]: Seems those changes were overwritten by an update... I've put them back now.

**Review by olejandro — APPROVED** [review]

**Pull request merged**